src/Simulation/qdk_sim_rs/NuGet.Config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
```

Comment:
I realize this is the latest advice for how to avoid nuget injection attacks, but it also means that this folder will not have access to the qdk-alpha feed configured at the root which may be surprising and lead to confusion later. Was there specific motivation behind including this here?

Reply:
This is used only for restoring the `dotnet-script` tool; ideally, that shouldn't be looking at the qdk-alpha feed, so that it always gets the right version off of the public nuget.org feed.